operator/deploy/crd.yaml

```diff
@@ -30,11 +30,335 @@ spec:
+                  type: object
+                  properties:
+                    accessible_namespaces:
+                      type: array
+                      items:
+                        type: string
+                    additional_service_yaml:
+                      type: object
+                      additionalProperties: true
+                    affinity:
+                      type: object
+                      properties:
+                        affinity:
+                          type: object
+                          additionalProperties: true
+                        pod:
+                          type: object
+                          additionalProperties: true
+                        pod_anti:
+                          type: object
+                          additionalProperties: true
+                    image_name:
+                      type: string
+                    image_pull_policy:
+                      type: string
+                      enum:
+                        - Always
+                        - Never
+                        - IfNotPresent
+                    image_pull_secrets:
+                      type: array
+                      items:
+                        type: string
+                    image_version:
+                      type: string
+                    ingress_enabled:
+                      type: boolean
+                      default: true
+                    namespace:
+                      type: string
+                    secret_name:
+                      type: string
+                    service_type:
+                      type: string
+                      enum:
+                        - NodePort
+                        - ClusterIP
+                        - LoadBalancer
+                        - ExternalName
+                    tolerations:
+                      type: array
+                      items:
+                        type: object
+                        additionalProperties: true
+                    verbose_mode:
+                      type: number
+                      minimum: 0
+                      maximum: 5
+                    version_label:
+                      type: string
+                    view_only_mode:
+                      type: boolean
+                external_services:
+                  type: object
+                  properties:
+                    grafana:
+                      type: object
+                      properties:
+                        auth:
+                          type: object
+                          properties:
+                            ca_file:
+                              type: string
+                            insecure_skip_verify:
+                              type: boolean
+                            password:
+                              type: string
+                            token:
+                              type: string
+                            type:
```

Comment:
type should be an enum - it accepts 3 values - `basic`, `bearer`, and `none` - see: https://github.com/kiali/kiali/blob/v1.9/config/config.go#L47-L50